configs/benchmark/values.yaml

```diff
@@ -0,0 +1,8 @@
+wrk2:
+  duration: 120
+  connections: 96
+  RPS: 3000
+  serviceMesh: ""
+  app:
+    name: bookinfo
+    count: "10"
```

Comment:
Is there a way for Helm to auto-detect this? The benchmark script used to do something akin to
```
$ kubectl get namespaces | grep -c bookinfo
```

Reply:
I don't think this is doable at the helm templating level. This has to be taken care of by the user/script that installs benchmarking app.